IFC4x3/PropertyConstants/r/ROUNDABOUTLINE_3weMGt1k1ERe5JxBFCXZa2.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="ROUNDABOUTLINE_3weMGt1k1ERe5JxBFCXZa2" Name="ROUNDABOUTLINE" UniqueId="faa16437-06e0-4e6e-8153-ecb3cc863902">
+	<Documentation>Round about line.</Documentation>
```

Comment:
Is is "roundabout" or "round about"?

Reply:
You are right. Updated: https://github.com/bSI-InfraRoom/IFC-Specification/pull/178/commits/028e8e83605d724326e1c6af7cf041c587b9c877